src/Snapshooter/SnapshotAssert.cs

```diff
@@ -88,13 +88,17 @@ public class SnapshotAssert : ISnapshotAssert
                         "rerun your tests.");
                 }
 
-                _snapshotFileHandler
-                    .SaveNewSnapshot(snapshotFullName, actualSnapshotSerialized);
-                return;
+                _snapshotFileHandler.SaveNewSnapshot(snapshotFullName, actualSnapshotSerialized);
+
+                savedSnapshotSerialized = _snapshotFileHandler.ReadSnapshot(snapshotFullName);
```

Comment:
I'm not sure how can I add test cases for this change. For the sake, I can use Moq to verify if `_snapshotFileHandler.ReadSnapshot` was called twice and `_snapshotComparer.CompareSnapshots` called once.

If anyone can guide me I can work on that.